test/e2e-olm/kibana_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	consolev1 "github.com/openshift/api/console/v1"
+	routev1 "github.com/openshift/api/route/v1"
```

Comment:
Make sure to use consistent import naming. You use `routev1` here but `route` in the other places.

Reply:
Fixed in the other places.